nvme-print.c

```diff
@@ -6346,8 +6346,8 @@ void nvme_show_list_item(nvme_ns_t n)
 	snprintf(format, sizeof(format), "%3.0f %2sB + %2d B", (double)lba,
 		l_suffix, nvme_ns_get_meta_size(n));
 
-	printf("%-21s %-20s %-40s %-9d %-26s %-16s %-8s\n",
-		nvme_ns_get_name(n), nvme_ns_get_serial(n),
+	printf("%-21s %-21s %-20s %-40s %-9d %-26s %-16s %-8s\n",
+		nvme_ns_get_name(n), nvme_ns_get_generic_name(n), nvme_ns_get_serial(n),
```

Comment:
Well, that will change the output as a new field is added in between existing ones. Which will break any parser there is.
I'm not that happy about it; our system management folks have the habit of parsing outputs, and do get terribly upset by these things.
Keith?

Reply:
Agreed on that point. but I think namespace generic device pretty much deserves to print out here along with other fields. Just like the --verbose mode, can we print it out on normal mode also ?  It would be great to print out where the I/O should   go through  (blkdev or chardev) for namespaces.

Reply:
And of course, it can be parsed well with a fixed format, but we have `--output-format=json` for the parsers.
(I missed to add generic device to json format, but we have this format for parsers)